src/components/Card/indexCard.js

```diff
@@ -47,6 +53,18 @@ export default function IndexCard({card}) {
 		<StyledCard>
 			<StyledCardHeader>
 				<StyledCardHeadline>{card.name}</StyledCardHeadline>
+				<BookmarkButton
+					onClick={() => {
+						bookmarked(card.id);
+						setIsBookmarked(!isBookmarked);
+					}}
+				>
+					{cards[card.id].isBookmarked ? (
+						<FilledBookmarkIcon id={card.id} />
+					) : (
+						<NotFilledBookmarkIcon id={card.id} />
+					)}
+				</BookmarkButton>
```

Comment:
As we mentioned you could think about using a single SVG component - but that's a task for "Future Chris" 😁